frontend/src/components/AppFooter.tsx

```diff
@@ -0,0 +1,119 @@
+'use client';
```

Comment:
I believe the footer, header, and layout can be server components if it's all static content?

Reply:
Yes, they should be. Please remove that for this PR if you can.

Reply:
Turns out since they're using `useState` they can't be RSC - need to be marked `use client` . Also I switched to just using the original Footer/Header for shared UI purposes described in the comment above and the tech spec